persdict.dat

```diff
@@ -62,3 +62,16 @@ sponte
 squacquerone
 subìto
 username
+snippet
+canvas
+WordPress
+popup
+LibreOffice
+evidenziazione
+color
```

Comment:
Questo in realtà è palese inglese, eviterei di inserirlo (che ne pensi? Secondo me è giusto che venga dato come errore in italiano).

Reply:
Hai ragione mi è sfuggito